pkg/controller/logstorage/logstorage_controller.go

```diff
@@ -464,8 +464,13 @@ func (r *ReconcileLogStorage) generateSecrets(
 	for _, certName := range certs {
 		cert, err := cm.GetCertificate(r.client, certName, common.OperatorNamespace())
 		if err != nil {
-			r.status.SetDegraded(operatorv1.ResourceReadError, "Failed to get certificate", err, log)
-			return nil, err
+			if certificatemanager.IsCertUsageError(err) {
```

Comment:
I think we need a big comment here explaining why we have this check, and why it's not needed everywhere else that `GetCertificate` is called